src/main/rx/fport.c

```diff
@@ -151,7 +153,7 @@ static void fportDataReceive(uint16_t c)
 
     const timeUs_t currentTimeUs = micros();
 
-    if (bufferPosition > 0 && cmpTimeUs(currentTimeUs, frameStartAt) > FPORT_TIME_NEEDED_PER_FRAME_US + 500) {
+    if (!skipUntilStart && (frameStarting || bufferPosition > 0) && cmpTimeUs(currentTimeUs, frameStartAt) > FPORT_TIME_NEEDED_PER_FRAME_US + 500) {
          reportFrameError(DEBUG_FPORT_ERROR_TIMEOUT);
 
```

Comment:
`frameStarting` is not reset - timeout after `STX` leaves receiver in 'frameStarting' state, leading to repeated timeouts for each character received